h/static/scripts/annotator/main.js

```diff
@@ -81,5 +73,6 @@ if (window.hasOwnProperty('hypothesisConfig')) {
 }
 
 Annotator.noConflict().$.noConflict(true)(function () {
-  window.annotator = new Klass(document.body, options);
+  window.annotator = new options.constructor(document.body, options);
+  delete options.constructor;
```

Comment:
:ghost:

Use a temporary variable and delete it `before` it's passed. That was the whole point of deleting it ;).